frotz.py

```diff
@@ -126,15 +131,45 @@ def start_game(self, game_file):
             # print a welcome banner and pause for a moment
             # that way the end user will have a chance to see which version of frotz we are using
             # and which file we are loading
-
-            if platform.machine().startswith('arm'):
-                logging.error('ARM not supported, yet') # FIXME
-                sys.exit(0)
+            if platform.version().find("Ubuntu") > -1 or platform.version().find("Debian") > -1:
```

Comment:
While this works for Ubuntu and Debian, a more correct method is to search for the binary in a list of paths, like how Linux does it.